LocalWiki.php

```diff
@@ -81,11 +82,18 @@ function onHtmlPageLinkRendererEnd(
 				return true;
 			}
 
-			$attribs['rel'] = 'nofollow';
+			$attribs[ 'rel' ] = 'nofollow';
```

Comment:
This is not needed, please remove the spaces to keep it consistent with the code style used here.

Reply:
No, the MediaWiki style guidelines say to use spaces like that.

Reply:
This is not Wikimedia, we don't have to use their code style. For examples of spaces not being used here, see https://github.com/miraheze/mw-config/blob/e7ca82a45d7b575c65ab97dc17a2d43c90c3bf22/LocalWiki.php#L6 https://github.com/miraheze/mw-config/blob/e7ca82a45d7b575c65ab97dc17a2d43c90c3bf22/LocalWiki.php#L21 and https://github.com/miraheze/mw-config/blob/e7ca82a45d7b575c65ab97dc17a2d43c90c3bf22/LocalWiki.php#L91 for a few examples.

Style consistency is important.

Reply:
Done